.github/workflows/cd.yml

```diff
@@ -24,6 +24,9 @@ jobs:
       - name: Install
         run: yarn --no-progress --non-interactive --no-lockfile
 
+      - name: Update locale files
+        run: python3 tools/refreshLocaleFiles.py

```

Comment:
Did it work? Do the ci logs show it working as expected?

Reply:
The script doesn't print anything. Must I be adding something on the lines of a verbose mode?

Reply:
The script now prints out messages as it updates locales.